src/resolvers/noderesolver.ts

```diff
@@ -1,24 +1,10 @@
-import Debug from "debug";
-import path from "path";
 import { ResolverContext, SubResolver } from ".";
-
-const debug = Debug("resolverengine:noderesolver");
+import { BacktrackFsResolver } from "./backtrackfsresolver";
 
 export function NodeResolver(): SubResolver {
-  return async (what: string, ctx: ResolverContext): Promise<string | null> => {
-    try {
-      let requirePaths: string[] = [];
-      let currentPath = ctx.cwd || process.cwd();
-      let lastPath: string = "";
-      do {
-        requirePaths.push(currentPath);
-        lastPath = currentPath;
-        currentPath = path.dirname(currentPath);
-      } while (currentPath !== lastPath);
-      return require.resolve(what, { paths: requirePaths });
-    } catch (e) {
-      debug("Node's require.resolve failed, returning null", e);
-      return null;
-    }
+  const backtrack = BacktrackFsResolver("node_modules");
```

Comment:
An edge case might happen if the package is in global node_modules

Reply:
Added to test excel

Reply:
I'm not sure we can handle a string that contains shell env variable?

Reply:
Or what was your intention for expected output?